config/rewrites.d/support.rackspace.com.json

```diff
@@ -960,8 +960,8 @@
         "status": 301
       },
       {
-        "description": "redirect Supported Operating Systems in Rackspace Public Cloud to new /how-to/supported-operating-systems-and-browsers-in-rackspace-public-cloud/",
-        "from": "^\\/how-to\\/supported-operating-systems-in-rackspace-public-cloud(.*)$",
+        "description": "redirect Supported Browsers to new /how-to/supported-operating-systems-and-browsers-in-rackspace-public-cloud/",
+        "from": "^\\/how-to\\/supported-browsers(.*)$",
```

Comment:
I would add this as a new redirect, so you don't break this one.  Add it right underneath or above this redirect.

Reply:
Are you saying to move the new redirect for Browsers under line 840, which is the one for supported OS?